beacon/goclient/signing.go

```diff
@@ -11,6 +12,21 @@ import (
 )
 
 func (gc *goClient) DomainData(epoch phase0.Epoch, domain phase0.DomainType) (phase0.Domain, error) {
+	if domain == types.DomainApplicationBuilder { // no domain for DomainApplicationBuilder. need to create.  https://github.com/bloxapp/ethereum2-validator/blob/v2-main/signing/keyvault/signer.go#L62
```

Comment:
is `DomainApplicationBuilder` created? if no is this code used?

Reply:
It's created in spec and used for signing in `ValidatorRegistrationRunner`. It's from https://github.com/bloxapp/ssv/pull/856 (this PR is based on it). It should be actually `spectypes`, not `types`